### PR TITLE
fix: default update thread for actions is EDT (#218)

### LINKED ISSUE #218: Override 'getActionUpdateThread' and choose 'EDT' or 'BGT'
Our Actions all have a warning that is causing a "fatal IDE error" when using `IC-2024.1`

> Override 'getActionUpdateThread' and choose 'EDT' or 'BGT'

See https://github.com/redhat-developer/intellij-openshift-connector/issues/738

### PR DESCRIPTION
**Pull request by adietish**
fixes #218

### CODE REVIEW

File: .github/workflows/IJ.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IC-2021.1, IC-2021.2, IC-2021.3, IC-2022.1, IC-2022.2, IC-2022.3, IC-2023.1, IC-2023.2]
+        IJ: [IC-2022.3, IC-2023.1, IC-2023.2, IC-2024.1]
```
Comment (sbouchet) [inline .github/workflows/IJ.yml:17]: means all dependent extensions using next version need to  validate this range version too ?

Comment (adietish) [reply .github/workflows/IJ.yml:17]: I'd think so, yes. Agree?

Comment (adietish) [reply .github/workflows/IJ.yml:17]: * intellij-kubernetes is at `IC-2022.1` (https://github.com/redhat-developer/intellij-kubernetes/blob/1f4637041eb0a0b36fb86a6589f1370f9406d702/.github/workflows/IJ.yml#L17)
* intellij-openshift-connector is at `IC-2022.1` (https://github.com/redhat-developer/intellij-openshift-connector/blob/30e4bc351d8df6a8058cf0d443706ab48cd2fed5/.github/workflows/validate_IJ_versions.yml#L12)
* intellij-tekton is at `IC-2022.2` (https://github.com/redhat-developer/intellij-tekton/blob/84f3d264c06b98290bcf88626d86fa8f5a085dd7/.github/workflows/IJ.yml#L17)
* intellij-knative is at `IC-2022.1` (https://github.com/redhat-developer/intellij-knative/blob/2e3c49bd684f27383a049c740e96f5e28a4f8ea4/.github/workflows/IJ.yml#L17)

Comment (sbouchet) [reply .github/workflows/IJ.yml:17]: agree to move up to at least 22.3 faor all downstream. not a big step.

**Review by sbouchet — APPROVED** [review]
LGTM

**Pull request merged**